repl/src/main/scala/ai/h2o/sparkling/repl/H2OInterpreter.scala

```diff
@@ -71,6 +74,15 @@ class H2OInterpreter(sparkContext: SparkContext, sessionId: Int) extends BaseH2O
 
     settings
   }
+
+  private def getJarsForShell(conf: SparkConf): String = {
+    val localJars = conf.getOption("spark.repl.local.jars")
+    localJars.map(_.split(",")).map(_.filter(_.nonEmpty)).toSeq.flatten
```

Comment:
Would it be possible to tune this piece of code a bit? Here we have a sequence of `map`, `map`, `flatten`, `map`. Would it be possible to use just `flatmap` with 0-1 `map` operations?

Reply:
Will do :)